x/farming/keeper/staking.go

```diff
@@ -405,40 +507,78 @@ func (k Keeper) Unstake(ctx sdk.Context, farmerAcc sdk.AccAddress, amount sdk.Co
 
 // ProcessQueuedCoins moves queued coins into staked coins.
 // It causes accumulated rewards to be withdrawn to the farmer.
```

Comment:
I think this comment needs to be updated since accumulated rewards will be withdrawn to the `UnharvestedRewardsReserveAcc`.

Reply:
That's right. I'll update the comment.